### ISSUE TITLE
Keyboard navigation between full-screen images

### ISSUE THREAD

**Issue by morellodev** [ant-design/ant-design#29323]
- [X] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
It would be a nice accessibility improvement if we could navigate between full-screen images of the `<Image.PreviewGroup>` component with the arrow keys.

### What does the proposed API look like?
A keyboard event handler should be added when an image is opened in full-screen mode.

**Comment by daya2940** [comment]
need help on this as how to implement this feature of keyboard navigation in antd using image preview

**Comment by QiuMing** [comment]
Hi ，is the feature is planning now  , it seems a high frequency  requirement . I try to find some way to implement it  ,but fail .
https://github.com/ant-design/ant-design/discussions/30092

**Comment by afc163** [comment]
trace #33363
